actors/builtin/paych/paych_actor.go

```diff
@@ -64,9 +65,9 @@ func (pca *Actor) Constructor(rt vmr.Runtime, params *ConstructorParams) *adt.Em
 // The account actor constructor checks that the embedded address is associated with an appropriate key.
 // An alternative (more expensive) would be to send a message to the actor to fetch its key.
```

Comment:
Please update the comment. The alternative is no longer relevant.

Reply:
This is done.